POLMED/src/components/button/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import "./style.css";
+
+const Button = () => {
```

Comment:
docelowo chcemy aby nasz komponent brał jakieś propsy, dzięki którym będziemy mogli manipulować jego contentem oraz kolorem lub tez szerokością

dodaj na ten moment propsy, które będą odpowiadać za kolor oraz kontent i na ich podstawie ustawiaj odpowiednio kolor i kontent - jak cos to pisz do mnie na priv